scripted/sbt/src/main/scala/sbt/test/SbtHandler.scala

```diff
@@ -80,7 +80,10 @@ final class SbtHandler(directory: File,
     val thread = new Thread() { override def run() = { p.exitValue(); server.close() } }
```

Comment:
```
[error] x compiler-project / inc-pickled-refinement compiler-project / inc-pickled-refinement
[error]  Cause of test exception: {line 7}  Command failed: Remote sbt initialization failed
```

The forked sbt on the other side is failing to get up, you might need more println around here where the Process is being spawned.